packages/pluggableWidgets/floating-action-button-native/src/FloatingActionButton.editorConfig.ts

```diff
@@ -0,0 +1,29 @@
+import { ImageProps, ContainerProps } from "@widgets-resources/piw-utils";
+import { FloatingActionButtonPreviewProps } from "../typings/FloatingActionButtonProps";
+import { FloatingActionButtonPrimarySVG } from "./assets/FloatingActionButtonPrimary";
+
+const rowLayoutContainer: ContainerProps = { type: "Container", children: [] };
+const rowLayoutImage: ImageProps = {
+    type: "image",
+    document: FloatingActionButtonPrimarySVG(),
+    data: "",
+    width: 56,
+    height: 56
+};
+
+export const getPreview = (values: FloatingActionButtonPreviewProps) => ({
```

Comment:
I would suggest to export as a function

Reply:
The datagrid was also using a constant here. I think for this widget it reduces the code a bit because we don't have to use `return` with the arrow function

Reply:
Okay, let's keep it aligned with data grid for now. We can always refactor this in the future.